dist/js/iosCheckbox.js

```diff
@@ -52,10 +52,10 @@
                     // Check if the ios checkbox is checked
                     if (ios_checkbox.hasClass("checked")) {
                         // Update state
-                        org_checkbox.prop('checked', true);
+                        org_checkbox.attr('checked','checked');
```

Comment:
This solves a problem that prevented the checkbox type input from getting checked status when clicking the button.

Reply:
👍